src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
I don't understand what you want to do... 
About the 'path' config key, the comment in middleware-pipeline.global.php says :
> literal path prefix to match middleware, will not execute if path does not match!